tests/binary-custom-main/BUILD

```diff
@@ -1,13 +1,19 @@
 load(
     "@io_tweag_rules_haskell//haskell:haskell.bzl",
+    "haskell_import",
     "haskell_test",
 )
 
 package(default_testonly = 1)
 
+haskell_import(
+    name = "base_pkg",
```

Comment:
Out of curiosity, why not
```
haskell_import(
    name = "base",
)
```
?

Reply:
I have copied what @nmattia did for `binary-simple`, really, so no particular reason.

Reply:
and I copied it from the docs...